app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/DevelopmentPreviewWrapper.kt

```diff
@@ -24,13 +30,18 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.babylon.wallet.android.LinkConnectionStatusObserver.LinkConnectionsStatus
 import com.babylon.wallet.android.R
 import com.babylon.wallet.android.designsystem.theme.RadixTheme
 import com.babylon.wallet.android.designsystem.theme.RadixWalletTheme
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.persistentListOf
 
 @Composable
 fun DevelopmentPreviewWrapper(
     modifier: Modifier = Modifier,
+    linkConnectionsStatus: LinkConnectionsStatus? = null,
```

Comment:
I would change the implementation a bit so as to 
1. Show the statuses in mainnet too while in debug mode
2. Do not change the appearance of the dev status bar if the statuses are visible

```
@Composable
fun DevelopmentPreviewWrapper(
    modifier: Modifier = Modifier,
    linkConnectionsStatus: LinkConnectionsStatus? = null,
    devBannerState: DevBannerState = DevBannerState(false),
    content: @Composable (PaddingValues) -> Unit
) {
    Box(modifier = modifier) {
        var bannerHeight by remember { mutableStateOf(0.dp) }
        CompositionLocalProvider(LocalDevBannerState provides devBannerState) {
            content(if (devBannerState.isVisible || linkConnectionsStatus != null) PaddingValues(top = bannerHeight) else PaddingValues())
        }

        val density = LocalDensity.current
        if (devBannerState.isVisible) {
            Column(
                modifier = Modifier
                    .fillMaxWidth()
                    .background(RadixTheme.colors.orange2)
                    .statusBarsPadding()
                    .onGloballyPositioned { coordinates ->
                        bannerHeight = with(density) { coordinates.size.height.toDp() }
                    }
                    .padding(RadixTheme.dimensions.paddingXSmall)
            ) {
                Text(
                    modifier = Modifier.fillMaxWidth(),
                    text = stringResource(R.string.common_developerDisclaimerText),
                    style = RadixTheme.typography.body2HighImportance,
                    color = Color.Black,
                    textAlign = TextAlign.Center,
                )

                if (linkConnectionsStatus != null) {
                    LazyRow(
                        modifier = Modifier,
                        horizontalArrangement = Arrangement.spacedBy(RadixTheme.dimensions.paddingSmall)
                    ) {
                        items(items = linkConnectionsStatus.currentStatus()) { color ->
                            Canvas(
                                modifier = Modifier.size(12.dp),
                                onDraw = {
                                    drawCircle(color = color)
                                }
                            )
                        }
                    }
                }
            }
        } else if (linkConnectionsStatus != null) {
            LazyRow(
                modifier = Modifier
                    .statusBarsPadding()
                    .onGloballyPositioned { coordinates ->
                        bannerHeight = with(density) { coordinates.size.height.toDp() }
                    }
                    .padding(RadixTheme.dimensions.paddingXSmall),
                horizontalArrangement = Arrangement.spacedBy(RadixTheme.dimensions.paddingSmall)
            ) {
                items(items = linkConnectionsStatus.currentStatus()) { color ->
                    Canvas(
                        modifier = Modifier.size(12.dp),
                        onDraw = {
                            drawCircle(color = color)
                        }
                    )
                }
            }
        }
    }
}
```

Reply:
big thanks for this!